application/features/SFTP.py

```
@@ -100,9 +105,11 @@
             mode = self.sftp.stat(fullpath).st_mode
             if stat.S_ISREG(mode):
                 # print(fullpath, 'is file')
+                logger.debug(f"SFTP: {fullpath} is a file")
```

Ruff

Logging statement uses f-string

(G004)